samples/bookinfo/src/details/Gemfile.lock

```diff
@@ -12,4 +12,4 @@ DEPENDENCIES
   webrick (~> 1.7)
 
 BUNDLED WITH
-   2.5.10
+   2.5.3
```

Comment:
This matches the PR to main branch, but was this downgrade intended? Any particular reason?